### PR TITLE
PHP 8.2 Support

### PR DESCRIPTION
**Pull request by rogervila**
Hi,

This PR needs https://github.com/doctrine/DoctrineModule to release a 5.3.x version to support PHP 8.2

### CODE REVIEW

**Review by driehle — CHANGES_REQUESTED** [review]
Please take a look at the Github workflows and add 8.2 to continues integration and change coding standards and static analysis from 8.1 to 8.2.

File: psalm.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <psalm
     errorLevel="4"
-    phpVersion="8.0"
+    phpVersion="8.2"
```
Comment (driehle) [inline psalm.xml]: Please drop this line, then Psalm will inflect the PHP version from `composer.json`

**Comment by rogervila** [comment]
We will have to wait until https://github.com/doctrine/DoctrineModule/pull/796 is merged to get green on 8.2

**Comment by rogervila** [comment]
@driehle I upgraded DoctrineModule, based on the conversation on https://github.com/doctrine/DoctrineModule/issues/798

**Comment by driehle** [comment]
I think before we step deeper into the static analysis issues, it would make sense to provide a PR doing the same as in https://github.com/doctrine/DoctrineModule/pull/792. Then, we can rebase this PR and solve the remaining issues.

**Comment by driehle** [comment]
@rogervila Could you please rebase on latest 5.3.x?

**Review by driehle — APPROVED** [review]

**Pull request merged**